### DISCUSSION TITLE
Feature Request: Define a standard ordering of keys in dashboard JSON model

### DISCUSSION THREAD

**Discussion by jsenko** [grafana/grafana#48823] [General]
We use GitOps to define and maintain our Grafana dashboards. When performing updates, a common procedure is to modify a copy of the existing dashboard, export the new model and commit the changes into git so they can be persisted and deployed. The issue I commonly encounter is that the old and the new dashboards are difficult to compare before committing, since the order of the JSON keys in the model is not defined. The changes would be much easier to see and keep track of if the keys are always kept in e.g. alphabetical order when exported.

**Comment by jsenko** [comment]
Never mind, it seems the keys are already being ordered? Is the same true for the ordering of the panels list? I would be great if the panels were ordered by `id`.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
